packages/btc/README.md

```diff
@@ -225,6 +225,30 @@ const psbt = await sendRgbppUtxos({
 });
 ```
 
+### Construct a Full-RBF transaction
+
+```typescript
+import { sendRbf, networkTypeToConfig, DataSource, Collector, NetworkType } from '@rgbpp-sdk/btc';
+import { BtcAssetsApi } from '@rgbpp-sdk/service';
+
+const networkType = NetworkType.TESTNET;
+const config = networkTypeToConfig(networkType);
+
+const service = BtcAssetsApi.fromToken('btc_assets_api_url', 'your_token');
+const source = new DataSource(service, networkType);
+
+const psbt = await sendRbf({
+  txHex: 'your_original_transaction_hex',
+  from: account.address,
+  feeRate: 40, // the feeRate should be greater than the feeRate of the original transaction
+  changeIndex: 1, // optional, return change to outputs[changeIndex], will create a new output if not specified
+  changeAddress: 'address_to_return_change', // optional, where should the change satoshi be returned to
+  requireValidOutputsValue: false, // optional, default to false, require each output's value to be >= minUtxoSatoshi
+  requireGreaterFeeAndRate: true, // optional, default to true, require the fee rate&amount to be greater than the original transction
+  source,
+});
```

Comment:
If the replaced BTC transaction is bound with a `rgbpp_ckb_tx_virtual`, then the new isomophic transactions (new_btc_tx + rgbpp_ckb_tx_virtual) should be submitted to the `RGB++ CKB Transaction Queue`.

Do you think the example code should include this additional opertation?

Reply:
If you mean adding cases in the `examples/rgbpp` folder, then yes, I think it'll help users better understand how it works in a bigger picture.

Reply:
A related issue has been created:
- https://github.com/ckb-cell/rgbpp-sdk/issues/230